CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log - AWS SDK for Android
 
+## [Release 2.16.1](https://github.com/aws/aws-sdk-android/releases/tag/release_v2.16.1)
+
+### New Features
+
+- **AWS IoT**
+  - AWS IoT now depends on the paho version 1.2.2
```

Comment:
AWS Android SDK for IoT now depends on the `1.2.2` version of the Paho Library `org.eclipse.paho.client.mqttv3`.